tests/playwright/jest-playwright.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	launchOptions: {
+		headless: false
```

Comment:
This should not be hardcoded. We should essentially set an environment variable which if not exists runs in headless by default